core/src/test/kotlin/regression/CacheMissRegression.kt

```diff
@@ -81,7 +81,7 @@ object FakeGateway : Gateway {
 }
 
 class CrashingHandler(val client: HttpClient, override val token: String) : RequestHandler {
-    override suspend fun <B : Any, R> handle(request: Request<B, R>): R {
+    override suspend fun <B : Any, R> handle(request: Request<B, R>, context: Any?): R {
```

Comment:
the huge problem with this context parameter is that we have to propagate it through all the way (from core to here)